etrago/cluster/electrical.py

```diff
@@ -103,8 +147,14 @@ def adjust_no_electric_network(etrago, busmap, cluster_met):
     # eHV network
     busmap2 = {}
 
+<<<<<<< HEAD
+    # Map crossborder AC buses in case that they were not part of the 
+    # k-mean clustering
+    if not (etrago.args["network_clustering"]["cluster_foreign_AC"]) & (
+=======
     # Map crossborder AC buses in case that they were not part of the k-mean clustering
     if (not etrago.args["network_clustering"]["cluster_foreign_AC"]) & (
+>>>>>>> cb2d378071a6d314386a62194d10b9fe25a021bc
```

Comment:
There is still this unsolved merge conflict, can someone solve it? I don't know how it should look like.